### DISCUSSION TITLE
Is there any way to parallelize xagg?

### DISCUSSION THREAD

**Discussion by jsillin** [ks905383/xagg#69] [Q&A]
Wasn't sure if this was a hidden feature or not implemented yet. Curious how easy (or not) people think it might be. Looking to get precip data down to county level and it's taking roughly an hour for each timestep utilizing one thread.

**Comment by ks905383** [comment]
Hey @jsillin - so sorry, somehow did not see this!

There now is as of v0.3.3. Running xagg with 
`with xa.set_options(impl='numba')` 

supports parallelization using dask. It does not yet return dask arrays (i.e., `xa.aggregate()` force computes the aggregation before returning things through `agg.to_dataset()` - see the discussion in #93 ), but it definitely allows for parallelized workflows.
